website/notifications/tasks.py

```diff
@@ -31,10 +31,14 @@ def send_users_email(send_type):
         sorted_messages = group_by_node(info)
         if sorted_messages:
             if not user.is_disabled:
+                # If there's only one node in digest we can show it's preferences link in the template.
```

Comment:
Thanks for documenting this!

Super minor nit: `can_change_node_preferences` is really based on whether or not `node` exists, not the other way around. I think something like this would be more clear to future devs:
```
notification_nodes = sorted_messages['children'].keys()
node = AbstractNode.load(notification_nodes[0]) if len(notification_nodes) == 1 else None
mails.send_mail(
    ...
    can_change_node_preferences=bool(node)
)
```
Note: the logic is exactly the same in this implementation.